xstate-chatbot/nodejs/src/machine/messages/vitals.js

```diff
@@ -317,11 +334,51 @@ const messages = {
       },
     }
   },
+  fatehkit: {
+    prompt: {
+      en_IN: 'Fateh kit delivered ?',
+      pa_IN: 'ਫਤਿਹ ਕਿੱਟ ਮਿਲੀ ਹੈ?',
+      hi_IN: 'फतेह किट मिला?'
+    }
+},
+heartrelated: {
+  prompt: {
+    en_IN: 'Any Heart Related Problem ',
+    pa_IN: 'ਦਿਲ ਨਾਲ ਸਬੰਧਤ ਕੋਈ ਵੀ ਸਮੱਸਿਆ',
+    hi_IN: 'दिल से जुड़ी कोई भी समस्या'
+  }
+},
+kidneyrelated: {
+  prompt: {
+    en_IN: 'Any Kidney Related Problem  ',
+    pa_IN: 'ਕਿਡਨੀ ਨਾਲ ਸੰਬੰਧਤ ਕੋਈ ਵੀ ਸਮੱਸਿਆ',
+    hi_IN: 'किडनी से जुड़ी कोई भी समस्या'
+  }
+},
+cancerrelated: {
+  prompt: {
+    en_IN: 'Cancer ',
+    pa_IN: 'ਕੈਨਸਰ ',
+    hi_IN: 'कैंसर'
+  }
+},
   addVitals: {
     en_IN: 'Your vitals have been registered successfully. Our team from health department is monitoring you vitals and in case of need they will contact you . You can register vitals again whenever there is an important change. You may also contact state helpline 104   or the district control room at numbers mentioned below',
     pa_IN: 'ਤੁਹਾਡੇ ਮਹੱਤਵਪੂਰਨ ਲੱਛਣ ਸਫਲਤਾਪੂਰਵਕ ਰਜਿਸਟਰ ਹੋ ਗਏ ਹਨ.. ਸਿਹਤ ਵਿਭਾਗ ਦੀ ਸਾਡੀ ਟੀਮ ਇਹਨਾ ਦੀ ਨਿਗਰਾਨੀ ਕਰ ਰਹੀ ਹੈ ਅਤੇ ਲੋੜ ਪੈਣ \'ਤੇ ਉਹ ਤੁਹਾਡੇ ਨਾਲ ਸੰਪਰਕ ਕਰਨਗੇ. ਜਦੋਂ ਵੀ ਕੋਈ ਮਹੱਤਵਪੂਰਣ ਤਬਦੀਲੀ ਹੁੰਦੀ ਹੈ ਤਾਂ ਤੁਸੀਂ ਦੁਬਾਰਾ ਲੱਛਣ ਰਜਿਸਟਰ ਕਰ ਸਕਦੇ ਹੋ. ਤੁਸੀਂ ਸਟੇਟ ਹੈਲਪਲਾਈਨ 104 ਜਾਂ  ਹੇਠਾਂ ਦੱਸੇ ਨੰਬਰਾਂ \'ਤੇ ਜ਼ਿਲ੍ਹਾ ਕੰਟਰੋਲ ਰੂਮ ਨਾਲ ਵੀ ਸੰਪਰਕ ਕਰ ਸਕਦੇ ਹੋ',
     hi_IN: 'आपके महत्वपूर्ण लक्षण सफलतापूर्वक दर्ज कर लिए गए हैं।स्वास्थ्य विभाग की हमारी टीम इनकी निगरानी कर रही है और जरूरत पड़ने पर वे आपसे संपर्क करेंगे। जब भी कोई महत्वपूर्ण परिवर्तन होता है तो आप लक्षणों को फिर से पंजीकृत कर सकते हैं। आप राज्य हेल्पलाइन 104 या नीचे दिए गए नंबरों पर जिला नियंत्रण कक्ष से भी संपर्क कर सकते हैं।',
-  }
+  },
+  noUserFound:{
```

Comment:
I think these 2 states are near the start of the flow. If so place it wherever along the flow the question is being asked.

Reply:
In RRT flow if com_status=1 then heart problem , lung problem ,cancer issues are supposed to be  asked before adding vitals otherwise if com_status !=1 then directly jump to addvitals without asking above questions

Reply:
updated usernotfound and patient's number